docs/site/Creating-components.md

```diff
@@ -28,10 +35,22 @@ When a component is mounted to an application, a new instance of the component
 class is created and then:
 
 - Each Controller class is registered via `app.controller()`,
-- Each Provider is bound to its key in `providers` object.
+- Each Provider is bound to its key in `providers` object via
+  `app.bind(key).toProvider(providerClass)`
+- Each Class is bound to its key in `classes` object via
+  `app.bind(key).toClass(cls)`
+- Each Binding is added via `app.add(binding)`
+
+Please note that `providers` and `classes` are shortcuts for provider and class
+`bindings`.
 
 The example `MyComponent` above will add `MyController` to application's API and
-create a new binding `my-value` that will be resolved using `MyValueProvider`.
+create the following bindings to the application context:
```

Comment:
I am not sure if `create ... to` is grammatically correct. I think `create ... in` or `add ... to` would be better.
```suggestion
create the following bindings in the application context:
```

ping @bschrammIBM @dhmlau